tests/test_base_group.py

```diff
@@ -232,3 +232,17 @@ async def test_release_authorization(self) -> None:
 
             assert authorize_csc.authorized_users == f"-{self.basegroup.get_identity()}"
             assert authorize_csc.cscs_to_change == ",".join(self.basegroup.components)
+
+    async def test_show_auth_status(self) -> None:
+        async with self.make_group(usage=Usages.DryTest):
+            await self.basegroup.show_auth_status()
```

Comment:
It would probably be a good idea to test that it logs what you expect. You can do something like this:

```
with self.assertLogs(self.basegroup.log, level=logging.INFO) as base_group_logs:
    await self.basegroup.show_auth_status()

assert "???" in base_group_logs.output
```

But replace the `"???"` with the expected text.

Also, there might be a `pytest` variant for the log assertion but I don't remember it from the top of my head.